config/health_care_application.yml

```diff
@@ -0,0 +1,30 @@
+endpoints:
```

Comment:
How about you keep just dev/test/production sections in this file, and have production set `endpoint` from an ENV variable, dev/test can continue to set inline as they are now. Then don't have to do impactful change to application.yml